test/functional/setup/create_logins_azure.sql

```diff
@@ -0,0 +1,18 @@
+--for this script to work in Azure, use sqlcmd to connect to master database 
+IF NOT EXISTS (SELECT name FROM sys.sql_logins WHERE name = 'test_password')
+BEGIN
+    CREATE LOGIN test_password WITH PASSWORD='! ;4triou';
```

Comment:
what are those tests dependent on this pass? You can generate the pass in a parent setup script and then pass it on to dependent scripts